src/components/PlaylistModal/PlaylistModal.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useReducer } from "react";
+import { usePlaylist } from "../../contexts";
+import { playlistModalReducer } from "../../reducers";

```

Comment:
Not mandatory but good to know: You can use path alias to avoid moving back in folders using ../../
refer For Create React App without using Craco section in this https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160

Reply:
Yes, I will try that.